src/main/java/com/gam/api/service/work/WorkServiceImpl.java

```diff
@@ -64,40 +71,31 @@ public WorkResponseDTO createWork(Long userId, WorkCreateRequestDTO request) {
     public WorkResponseDTO deleteWork(Long userId, WorkDeleteRequestDTO request) {
         val workId = request.workId();
 
-        val user = userRepository.getUserById(userId)
-                .orElseThrow(() -> new EntityNotFoundException(ExceptionMessage.NOT_FOUND_USER.getMessage()));
-
-        if (user.getWorks().size() == 1) {
-            user.deleteLinks();
-            user.updateUserStatus(UserStatus.NOT_PERMITTED);
-        }
+        val user = findUser(userId);
 
-        val work = workRepository.getWorkById(workId)
-                .orElseThrow(() -> new EntityNotFoundException(ExceptionMessage.NOT_FOUND_WORK.getMessage()));
+        val work = findWork(request.workId());
 
         if (!isOwner(work, userId)) {
             throw new WorkException(ExceptionMessage.NOT_WORK_OWNER.getMessage(), HttpStatus.BAD_REQUEST);
         }
 
-        val isFirstWork = work.isFirst();
+        val wasFirst = work.isFirst();
+        val workCount = user.getActiveWorks().size();
+        work.setActive(false);
+        work.setIsFirst(false);
 
-        val photoUrl = work.getPhotoUrl();
-
-        if (Objects.equals(photoUrl, "")) {
-            throw new WorkException(ExceptionMessage.WORK_NO_THUMBNAIL.getMessage(), HttpStatus.BAD_REQUEST);
+        if (workCount == 1) { // 작업물이 한개 였을 때
+            user.setSelectedFirstAt(null);
+            user.setWorkThumbNail(null);
+            user.setFirstWorkId(null);
```

Comment:
```suggestion
    user.setFirstWorkId(null);
    user.updateUserStatus(UserStatus.NOT_PERMITTED);
```
staus 업로드 로직 다시 살려놔 주세요..!

Reply:
![스크린샷 2024-01-20 오후 12 53 32](https://github.com/Gam-develop/gam-server/assets/77230391/5f323e62-29ec-4fae-aa38-5f4c1b8fa958)

Reply:
넵!